doc-examples/hibernate-reactive-example-java/src/test/java/example/ProductRepositoryTest.java

```diff
@@ -53,19 +53,4 @@ void testProgrammaticTransactions() {
                 productManager.find("Watch").block().getName()
         );
     }
-
-    @Test
-    void testFindCaseInsensitive() {
```

Comment:
Why this test is deleted?

Reply:
Because MySQL is not case sensitive when comparing `varchar` types.